UnrealGdkJobTemplate.yml

```diff
@@ -0,0 +1,43 @@
+# JOB LEVEL TEMPLATE:
```

Comment:
Template files should not be in this repo.
They should be here: https://dev.azure.com/PlayFabInternal/Main/_git/UnrealGdkTemplate